Stripe/PublicHeaders/STPPaymentOptionsViewController.h

```diff
@@ -103,11 +103,13 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  The Stripe ID of a payment method to display as the default pre-selected option.
```

Comment:
so do we not populate this with the value stored in NSUserDefaults?

Reply:
right, I'm worried that would be a breaking change.  It would behave a little weirdly if we populated it, it would be nil until the network call finishes.

I've half a mind to deprecate the property, too.

WDYT?

Reply:
hmm, maybe deprecate and make `retrieveLastSelectedPaymentMethodIDForCustomerWithCompletion` public?